### PR TITLE
Add recipe for HLA*LA

### PR DESCRIPTION
**Pull request by AlexanderDilthey**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/hla-la/1.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  url: https://github.com/DiltheyLab/HLA-LA/archive/v{{ version }}.tar.gz
+  sha256: 68e1398537c09b4ef5ac00eca10343e5a0ede70a29b2cfc9c4feefffa5bde57c
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} 
+    - jsoncpp
+  host:
+    - zlib
+    - boost
```
Comment (bgruening) [inline recipes/hla-la/1.0/meta.yaml]: boost-cpp is probably enough

File: recipes/hla-la/1.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+
+source:
+  url: https://github.com/DiltheyLab/HLA-LA/archive/v{{ version }}.tar.gz
+  sha256: 68e1398537c09b4ef5ac00eca10343e5a0ede70a29b2cfc9c4feefffa5bde57c
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} 
+    - jsoncpp
```
Comment (bgruening) [inline recipes/hla-la/1.0/meta.yaml]: jsoncpp can probably be removed from build

File: recipes/hla-la/1.0/build.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+find . -maxdepth 1 -mindepth 1 -not -name src -exec mv {} src/ \;
+
+cd src
+
+mkdir ../bin
+mkdir ../obj
+mkdir ../graphs
+
+mkdir ../localBamTools
+cd ../localBamTools
```
Comment (bgruening) [inline recipes/hla-la/1.0/build.sh]: You can use popd and pushd here.

File: recipes/hla-la/1.0/meta.yaml (hunk 1 above)
Comment (AlexanderDilthey) [reply recipes/hla-la/1.0/meta.yaml]: With only boost, tests fail.

File: recipes/hla-la/1.0/meta.yaml (hunk 2 above)
Comment (AlexanderDilthey) [reply recipes/hla-la/1.0/meta.yaml]: This is correct, thank you!

File: recipes/hla-la/1.0/build.sh (hunk 3 above)
Comment (AlexanderDilthey) [reply recipes/hla-la/1.0/build.sh]: Thanks! Modified accordingly.

**Comment by AlexanderDilthey** [comment]
Hi @bgruening, anything left to do here before we can merge?

**Comment by AlexanderDilthey** [comment]
Thank you!

**Comment by AlexanderDilthey** [comment]
@bgruening It seems that this recipe is not going into the nightly upload tests - or is it? I can see neither a red X nor a green check mark next to the commit in the commit history...

**Comment by bgruening** [comment]
This is annoying, CCI has not triggered a build as it seems. Sorry for that.
Let's try is out here: https://github.com/bioconda/bioconda-recipes/pull/13656

**Comment by AlexanderDilthey** [comment]
Hmm, still looks like it hasn't done anything, has it?

**Pull request merged**